dvc/data/_progress.py

```diff
@@ -30,3 +30,6 @@ def callback(self, phase, *args):
             # let Tqdm take care of when to update
             self.set_description_str(desc, refresh=False)
         self.update_to(completed, total)
+
+    def __del__(self):
+        return super().__del__()
```

Comment:
https://docs.python.org/3/reference/datamodel.html#object.__del__

> If a base class has a `__del__()` method, the derived class's `__del__()` method, **if any**, must explicitly call it to ensure proper deletion of the base class part of the instance.

Doesn't make sense to me how this fixes things.